MetaMask/metamask-extension
# feat: better storybook stories for the notification pages

## **Description**

This pull request enhances the Storybook stories for the notification pages. The improvements aim to provide better clarity and usability for developers working with notification components. The changes include more detailed and organized stories, making it easier to understand and test different notification states and components.

**### Important**

1. some of the components rendered by the new stories (`ui/pages/notification-details/notification-details.stories.tsx`) need to use an RPC endpoint to fetch certain data
2. to avoid the fetch using the Infura ID used by the application in PROD, this PR introduces an `INFURA_STORYBOOK_PROJECT_ID` env (`.metamaskrc.dist`)
3. If we’re in a Storybook environment, the `Infura-Source` header is not used in the utility (`ui/helpers/utils/notification.util.ts`)

## **Related issues**

N/A

## **Manual testing steps**

1. Open Storybook in your development environment
2. Navigate to the notification pages section
4. Review the updated stories for clarity and completeness
5. Verify that all notification states are represented and functioning as expected

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

https://github.com/user-attachments/assets/d8f91940-680b-46b6-97e6-0bda03bd54b9

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
